index.html

```diff
@@ -6,7 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Math Input</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.css">
-    <link rel="stylesheet" href="/mathquill.css">
```

Comment:
This isn't necessary anymore since we're include the mathquill.css as part of the math-input.css that we're building now.